tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
I think the main argument would be consistency.  However, the tool environment doesn't have to exactly track the non-tool environment.  And the differences in helpers we are talking about (for IE 8 compatibility) apply to edge cases that probably won't come up, especially in peer-reviewed Meteor code.

Separately from this review, I still think we should maintain the babel-runtime package for the app environment, though I admit I may have added more transforms without adding the corresponding helpers.  Once we release and things settle, and with a few more tests, the upkeep will be minimal.